src/actions/index.js

```diff
@@ -22,3 +23,9 @@ export const hoverOutAction = (cardId) => {
     id: cardId
   };
 };
+
+export const checkedJointAccount = () => {
+  return {
+    type: CHECKED
```

Comment:
That's a much better idea. Redux-form-material-ui even had the state toggling implemented for me.